### PR TITLE
fix: input-number inputFontSize

### PR DESCRIPTION
**Pull request by zombieJ**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix InputNumber component token `inputFontSizeSM` & `inputFontSizeLG` not working.       |
| 🇨🇳 Chinese |     修复 InputNumber 组件 token `inputFontSizeSM` 和 `inputFontSizeLG` 不生效的问题。     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input-number/style/index.ts [hunk 1]
```diff
@@ -75,6 +76,8 @@ const genInputNumberStyles: GenerateStyle<InputNumberToken> = (token: InputNumbe
     calc,
   } = token;
 
+  console.log('>>>>>', token.inputFontSize, token.inputFontSizeSM, token.inputFontSizeLG);
```
Comment (li-jia-nan) [inline components/input-number/style/index.ts:79]: console

File: components/input-number/demo/basic.tsx [hunk 2]
```diff
@@ -1,11 +1,75 @@
 import React from 'react';
 import type { InputNumberProps } from 'antd';
-import { InputNumber } from 'antd';
+import { ConfigProvider, Flex, Input, InputNumber, Radio } from 'antd';
 
-const onChange: InputNumberProps['onChange'] = (value) => {
-  console.log('changed', value);
-};
+class App extends React.Component {
```
Comment (li-jia-nan) [inline components/input-number/demo/basic.tsx:5]: 为啥用 class 大佬？

Comment (zombieJ) [reply components/input-number/demo/basic.tsx:5]: 忘记清理了，thx

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
这个要上车吗？

**Pull request merged**